zkevm-circuits/src/evm_circuit/execution/error_invalid_jump.rs

```diff
@@ -130,6 +140,15 @@ impl<F: Field> ExecutionGadget<F> for ErrorInvalidJumpGadget<F> {
             )
         });
 
+        cb.require_equal(
+            "rw_counter_end_of_reversion = rw_counter + reversible_counter - 1",
+            rw_counter_end_of_reversion.expr(),
+            cb.curr.state.rw_counter.expr()
+                + cb.rw_counter_offset()
+                + cb.curr.state.reversible_write_counter.expr()
+                - 1.expr(),
+        );
```

Comment:
I'd suggest this to be (as well as those in other exceptions):

```suggestion
        let rw_counter_end_of_step =
            cb.curr.state.rw_counter.expr() + cb.rw_counter_offset() - 1.expr();
        cb.require_equal(
            "rw_counter_end_of_reversion = rw_counter_end_of_step + reversible_counter",
            rw_counter_end_of_reversion.expr(),
            rw_counter_end_of_step + cb.curr.state.reversible_write_counter.expr(),
        );
```

To make the `-1` easier to realize.

Reply:
applied in [faa3721](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1030/commits/faa372133d9079ff17be8f7346c059016dacb82a)  more easier to read. thanks.